src/sentry/tasks/reprocessing.py

```diff
@@ -72,10 +74,10 @@ def batched_delete(model_cls, **filter):
         # out at that point and never do the delete.
         #
         # Better to delete a few rows than none.
-        while True:
+        for _ in range(MAX_BATCHES_PER_MODEL):
             # Django already loads this into memory, might as well do it
             # explicitly. Makes check for result emptiness cheaper.
-            result = set(model_cls.objects.filter(**filter)[:200].values_list("pk", flat=True))
+            result = set(model_cls.objects.filter(**filter)[:1000].values_list("pk", flat=True))
```

Comment:
Is `[:1000]` similar to SQL's limit? ORM makes it extremely hard to understand the database implications of this change

Reply:
Also shouldn't `:1000` be replaced with `MAX_BATCHES_PER_MODEL`?

Reply:
Yeah it just adds a `LIMIT 1000`. Fetching/deleting 1000 ids at a time should be fine here

Reply:
> Also shouldn't `:1000` be replaced with `MAX_BATCHES_PER_MODEL`?

No, that's the max number of time to make the query. The numbers happen to be the same but are separate concepts

Reply:
I see. Thank you for explaining. Would you mind adding a comment?